market/connector/google-geo-api/google-geo-api-demo/src_hd/google/getp/map/map.xhtml

```diff
@@ -0,0 +1,33 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/basic-8.xhtml">
+		<ui:define name="title">POC google geo api</ui:define>
+		<ui:define name="content">
+
+			<h3>
+			This is an POC for google geo api
```

Comment:
I'd not call it a POC: it lower the value of the product ... maybe it's rather a Demo?

Reply:
I changed it to Demo

Reply:
I would go for this text to be inline with the README.md: This is a demo for Axon Ivy's Google Maps API connector

Reply:
I added inside README.md